app/models/rapidfire/survey.rb

```diff
@@ -2,9 +2,18 @@ module Rapidfire
   class Survey < ActiveRecord::Base
     has_many  :questions
     validates :name, :presence => true
+    has_many :conditionals, dependent: :destroy
 
     if Rails::VERSION::MAJOR == 3
       attr_accessible :name, :introduction
     end
+
+    def get_conditional(question)
+      conditionals.each do |conditional|
```

Comment:
Refactor of `get_conditional`

`conditionals.detect { |conditional| conditional.questions.include?(question) }`